metrics-exporter/src/bin/io_engine/client/pool_stat.rs

```diff
@@ -0,0 +1,103 @@
+use crate::{client::grpc_client::GrpcClient, error::ExporterError};
+
+use serde::{Deserialize, Serialize};
+
+/// This stores IoStat information of a pool.
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub(crate) struct PoolIoStat {
+    name: String,
+    bytes_read: u64,
+    num_read_ops: u64,
+    bytes_written: u64,
+    num_write_ops: u64,
+    read_latency: u64,
+    write_latency: u64,
+}
+
+impl PoolIoStat {
+    /// Get name of the pool.
+    pub(crate) fn name(&self) -> &String {
+        &self.name
+    }
+
+    /// Get used bytes read of the pool.
+    pub(crate) fn bytes_read(&self) -> u64 {
+        self.bytes_read
+    }
+
+    /// Get total number of read ops of the pool.
+    pub(crate) fn num_read_ops(&self) -> u64 {
+        self.num_read_ops
+    }
+
+    /// Get the total bytes written in bytes.
+    pub(crate) fn bytes_written(&self) -> u64 {
+        self.bytes_written
+    }
+
+    /// Get total number of write ops of the pool.
+    pub(crate) fn num_write_ops(&self) -> u64 {
+        self.num_write_ops
+    }
+
+    /// Get total read latency in usec of the pool.
```

Comment:
let's postfix var and fn with _us to make it clearer

Reply:
Sure